profiles/marcokumu.json

```diff
@@ -0,0 +1,6 @@
+{
+    "image": "https://user-images.githubusercontent.com/90620737/181904436-f701dab8-5940-4572-951d-4c714cbca8ee.png",
+    "issueId": 1268,
+    "name": "Okumu Mark",
+    "username": "marcokumu"
+  }
```

Comment:
Missing an endline. Without this endline, there could be issues with some IDEs and configurations.
```suggestion
}

```


__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.gif)](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix&nbsp; | &nbsp;**Tags:** Inefficient